src/main/java/org/jabref/gui/mergeentries/newmergedialog/ThreeWayMergeView.java

```diff
@@ -119,7 +120,12 @@ private Field getFieldAtIndex(int index) {
     private void addRow(int fieldIndex) {
         Field field = getFieldAtIndex(fieldIndex);
 
-        FieldRowView fieldRow = new FieldRowView(field, getLeftEntry(), getRightEntry(), getMergedEntry(), fieldMergerFactory, fieldIndex);
+        FieldRowView fieldRow;
+        if (field == StandardField.AUTHOR) {
```

Comment:
Please check for `FieldProperty.PERSON_NAMES`, because this can happen for all fields containing a list of persons.